transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -174,9 +201,19 @@ protected void initChannel(Channel ch) throws Exception {
                 }
             }
         });
+        if (register) {
+            ChannelFuture future = loop.register(this);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "future". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)